episodes/03-working-with-MARC-files.md

```diff
@@ -173,11 +181,19 @@ If you change your preferences for the MarcEditor, the tool used to work with MA
 ## Reset settings for the MarcEditor in Preferences
 
 1. Click Edit → Preferences
-2. Select "MarcEditor" in the Preferences window in the left hand pane
-3. In the right hand pane, select Set Defaults for either font or font size
+2. Select "MarcEditor" in the Preferences window in the left-hand pane
+3. In the right-hand pane, select Set Defaults for either font or font size
```

Comment:
```suggestion
3. In the right-hand pane, under Set MarcEditor Font Properties, select Set Defaults
```
I checked and when clicking "set defaults" both the font size and font are returned to the default settings, even if only the font or font size is highlighted in the list